### PR TITLE
Add buttons for my presences

### PR DESCRIPTION
**Pull request by fantomitechno**
So I see that I can add buttons so I do it fastest I can !

> Proof it work : 
KaniShiel : 
image

Gunivers : 
image

image

Altearn : 
image

BuildTheEarth : 
image

image

FR-Minecraft : 
image

image

And that's it

### CODE REVIEW

**Comment by fantomitechno** [comment]
And I forgot to bump version, I'm doing it

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]
Resolve merging conflicts

**Comment by fantomitechno** [comment]
Done, idk why when I tried to merge upstream/master into my master branch it, just don't do anything

**Review by StrikerFRFX — APPROVED** [review]

File: websites/A/Altearn/presence.ts [hunk 1]
```diff
@@ -7,11 +7,18 @@ presence.on("UpdateData", async () => {
...
   presenceData.startTimestamp = browsingStamp;
   if (privacy) {
     presenceData.details = "Browsing";
   } else {
+  if(button) presenceData.buttons = [
+      {
+        label: "Visite Website",
+        url: "https://altearn.xyz/"
+      }
+    ]
```
Comment (ririxidev) [inline websites/A/Altearn/presence.ts:21]: image

File: websites/B/BuildTheEarth/presence.ts [hunk 2]
```diff
@@ -8,56 +8,72 @@ presence.on("UpdateData", async () => {
...
-    presenceData.details = "Viewing a page:";
-    presenceData.state = document.title.replace(" - BuildTheEarth", "");
-    if (window.location.pathname.length != 1)
-      presenceData.details = "Viewing a BuildTeam:";
+  if(button) presenceData.buttons = [
+      {
+        label: "Visite Website",
+        url: "https://buildtheearth.net/"
+      }
+    ]
```
Comment (ririxidev) [inline websites/B/BuildTheEarth/presence.ts:23]: image

File: websites/G/Gunivers/presence.ts [hunk 3]
```diff
@@ -87,12 +94,24 @@ presence.on("UpdateData", async () => {
     presenceData.smallImageKey = "workspace";
     presenceData.details = "Viewing a page:";
     presenceData.state = "Gunivers Workspace";
+  if(button) presenceData.buttons = [
+      {
+        label: "Visit Website",
+        url: "https://project.gunivers.net/"
+      }
+    ]
```
Comment (ririxidev) [inline websites/G/Gunivers/presence.ts:102]: image
If you want to keep this button, change label for it

File: websites/F/FR-Minecraft/presence.ts [hunk 4]
```diff
@@ -64,224 +66,291 @@ presence.on("UpdateData", async () => {
...
-    } else if (window.location.pathname.includes("launcher-fr-minecraft.php")) {
-      presenceData.details = "Viewing a page:";
-      presenceData.state = "FR-Minecraft launcher";
+    } else {
+    if(button) presenceData.buttons = [
+        {
+          label: "Visite Website",
+          url: "https://fr-minecraft.net/"
+        }
+      ]
```
Comment (ririxidev) [inline websites/F/FR-Minecraft/presence.ts:75]: image

**Review by ririxidev — CHANGES_REQUESTED** [review]
fix deepscan issues too

**Comment by fantomitechno** [comment]
I didn't see that in the guidelines when I check but if you say it, I update ^^

**Review by ririxidev — APPROVED** [review]
Looks fine for me 👍 ! Don't worry because we updated guidelines yesterday so you couldn't check it before :D

**Review by Bas950 — APPROVED** [review]

**Pull request merged**